background/get-addon-settings.js

```diff
@@ -5,9 +5,39 @@ import minifySettings from "../libraries/common/minify-settings.js";
  the versions separately. Current versions:
 
  - editor-dark-mode 2 (bumped in v1.23 twice)
- - editor-theme3 4 (last bumped in v1.29)
+ - editor-theme3 3 (last bumped in v1.32)
```

Comment:
TODO: was editor-theme3 ever at version 4? Doesn't seem like it. Worth double checking.

Reply:
Looks like it was never at version 4. See #5146